### ISSUE TITLE
[php7.3b3] Call to private method AbstractRestParser::parseXml() from context 'RestXmlParser'

### ISSUE THREAD

**Issue by PhilETaylor** [aws/aws-sdk-php#1628]
php 7.3 beta 3

Error deep down in the call stack when running method ->has($file)... 

Call to private method Aws\Api\Parser\AbstractRestParser::parseXml() from context 'Aws\Api\Parser\RestXmlParser'

![local-maintain myjoomla com_en_sites_confirmdelete_wmeoqz](https://user-images.githubusercontent.com/400092/45384854-1a798d00-b607-11e8-8e22-7c58063cf986.png)

**Comment by diehlaws** [comment]
Hi @PhilETaylor, thanks for reaching out to us about this. I was unable to reproduce this on a docker container using the `php:7.3.0beta3-apache` image. 

Can you give us some more details about the versions you're using for PHP and the the AWS SDK for PHP? Including a link to the exact PHP build used when receiving this error would be ideal. In addition to this, providing a more complete code sample would help us test the exact operations with which you're running into trouble.

**Comment by PhilETaylor** [comment]
Firstly sorry for the lack of helpful information in initial opening of this issue, was in a rush and did not have time to report it correctly. 

I tried to recreate this today, and build you a complete docker based test case with PHP 7.1 RC1 and failed to replicate it once. I attempted to replicate the Flysystem code in an isolated test case, and every test passed, with no exceptions thrown. 

Sorry for wasting your time. I must have hit some edge case somehow the other day. I'll run my test docker from the office again on monday (at home now on "other" mac) and as long as that passes (which it should, thats the point of docker builds!) then I'll forget this ever happened. 

I'm happy to close this. If I get this again I'll create a replicable test case.
